### PR TITLE
Scripts/EasternKingdoms: Use std::chrono::duration overloads of EventMap

### PR DESCRIPTION
**Pull request by Carbenium**
**Changes proposed:**

-  Scripts/EasternKingdoms: Use std::chrono::duration overloads of EventMap

**Target branch(es):** 3.3.5/master

- [x] 3.3.5
- [ ] master

**Issues addressed:** Contributes to #25012

**Tests performed:** build

### CODE REVIEW

File: src/server/scripts/EasternKingdoms/AlteracValley/boss_drekthar.cpp [hunk 1]
```diff
@@ -65,10 +65,10 @@ class boss_drekthar : public CreatureScript
         {
             Talk(SAY_AGGRO);
             events.ScheduleEvent(EVENT_WHIRLWIND, 1s, 20s);
-            events.ScheduleEvent(EVENT_WHIRLWIND2, urand(1 * IN_MILLISECONDS, 20 * IN_MILLISECONDS));
+            events.ScheduleEvent(EVENT_WHIRLWIND2, 1s ,20s);
```
Comment (Jildor) [inline src/server/scripts/EasternKingdoms/AlteracValley/boss_drekthar.cpp:68]: Ops

Comment (Carbenium) [reply src/server/scripts/EasternKingdoms/AlteracValley/boss_drekthar.cpp]: Fixed. Thanks!

File: src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp [hunk 2]
```diff
@@ -256,7 +256,7 @@ class instance_stratholme : public InstanceMapScript
                                 if (EncounterState[0] == IN_PROGRESS || EncounterState[0] == FAIL)
                                     break;
                                 EncounterState[0] = data;
-                                events.ScheduleEvent(EVENT_BARON_RUN, 2700000);
+                                events.ScheduleEvent(EVENT_BARON_RUN, 2700s);
```
Comment (Shauren) [inline src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp:259]: this is 45 min

Comment (Shauren) [inline src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp:259]:
```suggestion
                                events.ScheduleEvent(EVENT_BARON_RUN, 45min);
```

File: src/server/scripts/EasternKingdoms/ZulGurub/boss_arlokk.cpp [hunk 3]
```diff
@@ -226,7 +226,7 @@ class boss_arlokk : public CreatureScript
                                 DoCast(target, SPELL_MARK_OF_ARLOKK, true);
                                 Talk(SAY_FEAST_PROWLER, target);
                             }
-                            events.ScheduleEvent(EVENT_MARK_OF_ARLOKK, urand(120000, 130000));
+                            events.ScheduleEvent(EVENT_MARK_OF_ARLOKK, 120s , 130s);
```
Comment (Shauren) [inline src/server/scripts/EasternKingdoms/ZulGurub/boss_arlokk.cpp:229]:
```suggestion
                            events.ScheduleEvent(EVENT_MARK_OF_ARLOKK, 120s, 130s);
```

**Pull request merged**